recipes/dinamo/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "DiNAMO" %}
+{% set version = "1.0" %}
+{% set sha256 = "9530934528dd3bff1a0b4dce33f7e5c3fece939214cbdab2ca98b5e4ef13c2dd" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  sha256: {{ sha256 }}
+  url: https://github.com/bonsai-team/{{name}}/archive/V{{version}}.tar.gz
+  patches:
+    - Makefile.patch
+
+build:
+  skip: True # [osx]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - boost-cpp
```

Comment:
Please remove boost-cpp from the "build" section, it's fine in `host` and `run`.